includes/wikia/services/UserService.class.php

```diff
@@ -86,6 +92,35 @@ public function resetPassword( User $targetUser ) {
 		return $tempPass;
 	}
 
+	/**
+	 * Generates password reset token and sends it to user via email
+	 *
+	 * @param User $targetUser
+	 * @param string $returnUrl Url the user will be redirected to after setting new password
+	 * @param string $tokenContext Used to choose the correct email template, so far only "facebook" value causes to use
+	 *                             a dedicated facebook-disconnect email
+	 *
+	 * @return boolean true for success, false otherwise
+	 */
+	public function requestResetToken( User $targetUser, $returnUrl, $tokenContext ) {
+		/** @var HeliosClient $heliosClient */
+		$heliosClient = Injector::getInjector()->get( HeliosClient::class );
+		$result = $heliosClient->requestPasswordReset( $targetUser->getId(), $returnUrl, $tokenContext );
+
+		if (!empty( $result->success ) && $result->success) {
```

Comment:
This can just be `if ( !empty( $result->success ) ) {` as the second check doesn't add any value or safety to the condition. `empty` checks the variable exists and isn't false (or false-y as it includes empty strings, null, 0, etc.). Having `&& $result->success` only checks if `$result->success` is truthy, which it has to be if `!empty( $result->success )` (there isn't a situation in this case where `!empty( $result->success )` would be true and `&& $result->success` would be false). So, the second part of the condition would only be useful if strictly checking the value (i.e. `if ( !empty( $result->success ) && $result->success === 'whatever the value should be' )`).

Also, minor: missing spaces per conventions.